workers/unity/Packages/com.improbable.gdk.core/Logging/ILogger.cs

```diff
@@ -0,0 +1,9 @@
+using UnityEngine;
```

Comment:
Don't think we should have yet another logger interface. Instead we could consider making changes to ILogDispatcher